cartesi-rollups/tutorials/calculator.md

```diff
@@ -1,18 +1,18 @@
 ---
 id: calculator
-title: Build a Calculator dApp with Python
-tags: [calculator, dapps, python, tutorial]
+title: Build a calculator dApp with Python
+tags: [calculator, apps, python, tutorial]
```

Comment:
the tag here wouldn't be dapps, or dapp, etc?

Reply:
Why is this resolved?